src/golang/lib/engine/aq_engine.go

```diff
@@ -368,7 +369,7 @@ func (eng *aqEngine) PreviewWorkflow(
 		jobManager, err = job.GenerateNewJobManager(
 			ctx,
 			dbDAG.EngineConfig,
-			&dbDAG.StorageConfig,
+			&dbDAG.StorageConfig, // cgwu: is this a bug?
```

Comment:
What's this comment about? Is it still relevant?

Reply:
seems to be a false alarm.